quickjs-libc.c

```diff
@@ -3976,6 +3990,7 @@ void js_std_init_handlers(JSRuntime *rt)
     ts->exc = JS_UNDEFINED;
 
     JS_SetRuntimeOpaque(rt, ts);
+    JS_AddRuntimeFinalizer(rt, js_free_rt, ts);
```

Comment:
Can't you avoid this by using the system allocator like before?

Reply:
I could but this seems neater. Is there a reason to prefer plain malloc?

Reply:
No strong opinion, but we are somewhat contradicting the documentation you just added: `the runtime itself is no longer usable.` I would've understood calling any RT function (including the memory ones) is a no-no.

Reply:
Do as I say, not as I do, is what I always say :-)

quickjs and quickjs-libc are strongly integrated so I don't think there's a dichotomy here. The doc comment is for downstream users and it's best to keep it simple ("do X" instead of "do X unless Y")